vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/business/simulation/WebServiceEngine.java

```diff
@@ -104,14 +104,17 @@ public String launch(String proxyFileName, String userDN)
             throws
             java.rmi.RemoteException,
             javax.xml.rpc.ServiceException {
+		String strProxy = null;
```

Comment:
seems you used tab indentation.
we use 4-spaces indent in vip.
Optionnaly you can try to convince me to change to tab but you need arguments.

Reply:
I think you initialize the var here as a good practice to declare all local variable at the beginning of the function ?

Reply:
Have you verified that `null` is supported and does not trigger an error in the vip/moteur communication ?

Reply:
> I think you initialize the var here as a good practice to declare all local variable at the beginning of the function ?

Yes

Reply:
> Have you verified that `null` is supported and does not trigger an error in the vip/moteur communication ?

I've run few jobs on the vip portal and it all works well, the vip-errors.log and moteur_server.log were empty.